Sources/TensorFlow/Core/Runtime.swift

```diff
@@ -680,8 +689,13 @@ extension _ExecutionContext {
     // The execution mode is effectively encoded in the TensorOperation.
     // We can use this to switch between different execution modes.
     // TODO: Can we interop between modes?
-    public static func makeOp(_ name: String, _ nOutputs: Int) -> TFTensorOperation {
-        return TFE_Op(name, nOutputs)
+    @usableFromInline
+    static func makeOp(
```

Comment:
```suggestion
    static func makeOperation(
```

Reply:
Will send out a separate PR as this needs changes to bindings as well.